projects/batfish/src/test/resources/org/batfish/grammar/cisco/testrigs/ibgp-reject-routerid-match/configs/r2

```diff
@@ -0,0 +1,23 @@
+! boot system flash:EOS.swi
+hostname r2
+!
+interface Ethernet1
+   no switchport
+   ip address 10.24.0.2/24
+!
+interface Loopback0
+   ip address 2.0.0.1/32
+!
+ip routing
+!
+!
+router bgp 1
+   router-id 2.2.2.2
+   neighbor 10.24.0.4 remote-as 1
```

Comment:
Please use loopbacks to establish iBGP sessions. Add static routes to those loopbacks wherever they are referred to. The next hop should be the physical interface address of the neighbor.